### ISSUE TITLE
Can't import tensorcircuit in nightly build

### ISSUE THREAD

**Issue by liwt31** [tencent-quantum-lab/tensorcircuit#93]
## Issue Description

Can't import tensorcircuit in nightly build.
The `psutil` dependency is introduced in `readout_mitigation.py` but it is not included in `setup.py`.
In CI installing `qiskit` also installs `psutil` so the error will not be raised.

## How to Reproduce

```bash
conda create -n psu python=3.8
conda activate psu
pip install tensorcircuit-nightly
python -m tensorcircuit
```

### Error Output

```python
tensorflow is not installed, and sparse Hamiltonian generation utilities are disabled
Traceback (most recent call last):
  File "/data/home/weitangli/miniconda3/envs/psu/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/data/home/weitangli/miniconda3/envs/psu/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/data/home/weitangli/miniconda3/envs/psu/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/data/home/weitangli/miniconda3/envs/psu/lib/python3.8/site-packages/tensorcircuit/__init__.py", line 33, in <module>
    from . import results
  File "/data/home/weitangli/miniconda3/envs/psu/lib/python3.8/site-packages/tensorcircuit/results/__init__.py", line 2, in <module>
    from . import readout_mitigation
  File "/data/home/weitangli/miniconda3/envs/psu/lib/python3.8/site-packages/tensorcircuit/results/readout_mitigation.py", line 10, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'
```

**Comment by refraction-ray** [comment]
ah, thought it is a built-in package before, then in this case, I will move the import within the function to avoid the general user to install this package

**Comment by refraction-ray** [comment]
fixed by 7fab89a1c1e7879387a553200aa5e2b336223ea7
